app/src/main/java/fr/gaulupeau/apps/Poche/Poche.java

```diff
@@ -256,18 +245,25 @@ protected void onPause() {
     @Override
     protected void onDestroy() {
         super.onDestroy();
-        if (database != null) {
+        if (database.isOpen()) {
             database.close();
         }
+
+
     }
 
     private void updateUnread() {
         runOnUiThread(new Runnable() {
             public void run() {
                 ArticlesSQLiteOpenHelper helper = new ArticlesSQLiteOpenHelper(getApplicationContext());
-		getDatabase();
-                int news = database.query(ARTICLE_TABLE, null, ARCHIVE + "=0", null, null, null, null).getCount();
-                btnGetPost.setText(String.format(getString(R.string.btnGetPost), news));
+                getDatabase();
+                if (database.isOpen()) {
```

Comment:
only called if db access is open. should be ok after the modification in getDatabase()